helita/sim/bifrost.py

```diff
@@ -6,16 +6,20 @@
 from glob import glob
 import numpy as np
 from . import cstagger
-
+import numba
```

Comment:
Why import numba if you're not using it?

Reply:
Thanks, I forgot to remove that one. I just remove it.